src/pages/VideoCall/index.js

```diff
@@ -50,35 +50,24 @@ const VideoCallFrame = () => {
             cssText: dailyCssText
         })
 
-        let participants = daily.participants()
+        const participants = daily.participants()
 
         const eventActions = (event) => {
-            daily.localVideo() === true ? setCamOn(true) : setCamOn(false)
-            daily.localAudio() === true ? setAudioOn(true) : setAudioOn(false)
-
-            if (participants && Object.keys(participants).length < 2) {
-                setParticipantStatus(t('waiting-participant'))
-            } else if (participants && Object.keys(participants).length < 3) {
-                if (event && event.participant.local === false && event.participant.video === false) {
-                    setParticipantStatus(t('participant-cam-off'))
-                } else if (event && event.participant.local === false && event.participant.video === true) {
-                    setParticipantStatus('')
-                }
-            } else {
-                setParticipantStatus('')
-            }
+            setCamOn(daily.localVideo())
+            setAudioOn(daily.localAudio())
+
+            const participantsLength = Object.keys(participants).length
 
             if (participants) {
-                switch (Object.keys(participants).length) {
-                case 3:
-                    setParticipantNumber(3)
-                    break
-                case 4:
-                    setParticipantNumber(4)
-                    break
-                default:
-                    setParticipantNumber(1)
+                if (participantsLength < 2) {
+                    setParticipantStatus(t('waiting-participant'))
+                } else if (participantsLength === 2 && (event && !event.participant.local)) {
+                    setParticipantStatus(!event.participant.video ? t('participant-cam-off') : '')
+                } else {
+                    setParticipantStatus('')
```

Comment:
Define `setParticipantStatus` default value with `(t('waiting-participant'))`

We can write it more simply. No need to check if participants is defined

```
setParticipantNumber(participantsLength)

if (participantsLength === 2 && (event && !event.participant.local)) {
    setParticipantStatus(!event.participant.video ? t('participant-cam-off') : '')
}
```

Reply:
Thanks, done. Still kept a `participantsLength > 2` check because participantStatus is displayed only when there are 2 participants (not more, knowing that calls can welcome up to 4 participants for now).

Reply:
Ok